### ISSUE TITLE
Python 3.10.0rc1 gives DeprecationWarning

### ISSUE THREAD

**Issue by othalan** [ndparker/rjsmin#26]
Using rjsmin on Python v3.10.0 RC1, I get the following Deprecation Warning:

```
<ipython-input-3-05a7209be334>:102: DeprecationWarning: PyUnicode_FromUnicode(NULL, size) is deprecated; use PyUnicode_New() instead
```

Python: 3.10.0 RC1
OS: MacOS 11.2.1

**Comment by ndparker** [comment]
Thank you, I'll look into it.

**Comment by ndparker** [comment]
Fixed in 1.2.0.
